wg-node-lifecycle/charter.md

```diff
@@ -0,0 +1,163 @@
+  Improve the descheduling, availability and migration capabilities of today's application
+  workloads. Also explore the interactions with other eviction mechanisms.
+- Coordinate pod termination and issues around de/scheduling, preemption and eviction.
+- Improve the Graceful/Non-Graceful Node Shutdown and consider how this affects the node lifecycle.
+  To graduate the [Graceful Node Shutdown](https://github.com/kubernetes/enhancements/issues/2000)
+  feature to GA and resolve the associated node shutdown issues.
+- Improve the scheduling and pod/node autoscaling to take into account ongoing node maintenance and
+  the new disruption model/evictions. This includes balancing of the pods according to scheduling
+  constraints. 
+- Consider improving the pod lifecycle of DaemonSets and Static pods during a node maintenance.
+- Explore the cloud provider use cases and how they can hook in into the node lifecycle. So that the
+  users can use the same APIs or configurations across the board.
+- Migrate users of the eviction based kubectl-like drain (kubectl, cluster autoscaler, karpenter,
+  ...) and other scenarios to use the new unified node draining approach.
+- Explore possible scenarios behind the reason why the node was terminated/drained/killed and how to
+  track and react to each of them. Consider past discussions/historical perspective
+  (e.g. "thumbstones").
+
+### Out of scope
+
+- Implementing cloud provider specific logic, the goal is to have high-level API that the providers
+  can use, hook into, or extend.
+- Infrastructure provisioning, deprovisioning solution or physical infrastructure lifecycle
+  management solution.
+
+## Stakeholders
+
+- SIG Apps
+- SIG Architecture
+- SIG Autoscaling
+- SIG CLI
+- SIG Cloud Provider
+- SIG Cluster Lifecycle
+- SIG Network
+- SIG Node
+- SIG Scheduling
+- SIG Storage
+
+Stakeholders span from multiple SIGs to a broad set of end users,
+public and private cloud providers, Kubernetes distribution providers,
+and cloud provider end-users. Here are some user stories:
+
+- As a cluster admin I want to have a simple interface to initiate a node drain/maintenance without
+  any required manual interventions. I also want to be able to observe the node drain via the API
+  and check on its progress. I also want to be able to discover workloads that are blocking the node
+  drain.
+- To support the new features, node maintenance, scheduler, descheduler, pod autoscaling, kubelet,
+  and other actors want to use a new eviction API to gracefully remove pods. This would enable new
+  migration strategies that prefer to surge (upscale) pods first rather than downscale them. It
+  would also allow other users/components to monitor pods that are gracefully removed/terminated
+  and provide better behaviour in terms of de/scheduling, scaling and availability.
+- As a cluster admin, I want to be able to perform arbitrary actions after the node drain is
```

Comment:
This one I'd move up with the other cases where you're describing admin perspective.

Reply:
Done, also moved some of the other user stories.